### PR TITLE
Added My features .. yet to test though

### PR DESCRIPTION
**Pull request by ApolloDev0**
didn't test them yet as i couldn't get it working on my system also i edited the Ascii Text so yeah

### CODE REVIEW

**Comment by Nitesh-thapliyal** [comment]
what are the features?

**Comment by ApolloDev0** [comment]
screenshot with pyautogui

**Comment by Nitesh-thapliyal** [comment]
update requirement.txt also

File: Atom.py [hunk 1]
```diff
@@ -145,6 +146,17 @@ def takecommand():
             
         elif "bye" in query:
             speak("See you later")
+
+        elif 'screenshot' in query:
+            img = pyautogui.screenshot()
+            global val
+            val = 0
+            if sys.platform == 'linux':
```
Comment (Nitesh-thapliyal) [inline Atom.py:154]: AttributeError: 'builtin_function_or_method' object has no attribute 'platform'

File: Atom.py [hunk 2]
```diff
@@ -145,6 +146,17 @@ def takecommand():
...
             speak("See you later")
+
+        elif 'screenshot' in query:
+            img = pyautogui.screenshot()
+            global val
+            val = 0
+            if sys.platform == 'linux':
+                img.save("./AtomImages/image-" + str(val) + ".png")
+                val += 1
+            if sys.platform == 'windows':
```
Comment (Nitesh-thapliyal) [inline Atom.py:157]: sys is the alias of the system right?

**Pull request merged**